AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -23,6 +23,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 m.kariyer.net##.download-app-wrapper
+wiwo.de#$#body.c-app-install--hidden { padding-top: 0px !important; }
```

Comment:
Not needed for desktop

Reply:
True, but desktop should not have that class in the first place.

Reply:
We have directives at the end of the file :)